docs/ref/models/database-functions.txt

```diff
@@ -76,11 +76,11 @@ Usage examples::
     A Python value passed to ``Coalesce`` on MySQL may be converted to an
     incorrect type unless explicitly cast to the correct database type:
 
-    >>> from django.db.models.expressions import RawSQL
+    >>> from django.db.models import DateTimeField
+    >>> from django.db.models.functions import Cast
```

Comment:
Unrelated, but adding `, Coalesce` would make this snippet independent of the imports in the previous example.